src/compile.rs

```diff
@@ -746,8 +747,24 @@ test!(string_max => r#"
 // GPGPU
 
 test!(hello_gpu => r#"
-    export fn main = GPU().print();"#;
-    stdout_contains "GPU {";
+    export fn main {
+      let g = GPU();
+      let b = g.createBuffer(storageBuffer(), filled(2.i32(), 4));
+      let plan = GPGPU("
+        @group(0)
+        @binding(0)
+        var<storage, read_write> vals: array<i32>;
+
+        @compute
+        @workgroup_size(4)
```

Comment:
I'll eventually update this to provide controls to let you decide how many workgroups you want to run. Then this length doesn't need to be hardwired in here (rather, it can be 1 and you can dynamically set the number of workgroups by the length of the array instead of modifying the size of the workgroup)